tests/sweep_framework/sweeps/conv2d/short/conv2d_short_sweep.py

```diff
@@ -1583,6 +1583,372 @@
         ],
         "is_conv1d": [True],
     },
+    "tt-forge_sweep_conv2d": {
+        "input_specs": [
+            # Contains following params
+            # [batch_size, output_channels, input_channels, input_height, input_width, kernel_height, kernel_width, stride_x, stride_y, pad_x, pad_y, groups, bias, dilation, datatype]
+            [1, 1024, 1024, 14, 14, 3, 3, 1, 1, 1, 1, 1, False, 1, "bfloat16"],
```

Comment:
can we use our enum for data type instead of the string?

Reply:
done.